sdk/openai/azure-ai-openai-stainless/src/samples/java/com/azure/ai/openai/stainless/ResponsesImageBase64Example.java

```
@@ -0,0 +1,70 @@
+
+package com.azure.ai.openai.stainless;
+
+import com.azure.identity.AuthenticationUtil;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.openai.azure.AzureOpenAIServiceVersion;
+import com.openai.client.OpenAIClient;
+import com.openai.client.okhttp.OpenAIOkHttpClient;
+import com.openai.credential.BearerTokenCredential;
+import com.openai.models.ChatModel;
+import com.openai.models.responses.ResponseCreateParams;
+import com.openai.models.responses.ResponseInputImage;
+import com.openai.models.responses.ResponseInputItem;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Base64;
+import java.util.Collections;
+
+public final class ResponsesImageBase64Example {
+    private ResponsesImageBase64Example() {}
+
+    public static void main(String[] args) throws IOException {
+        // Configures using one of:
+        // - The `OPENAI_API_KEY` environment variable
+        // - The `AZURE_OPENAI_ENDPOINT` and `AZURE_OPENAI_KEY` environment variables
+        OpenAIClient client = OpenAIOkHttpClient.builder()
+            .azureServiceVersion(AzureOpenAIServiceVersion.latestPreviewVersion())
+            // Gets the API key from the `AZURE_OPENAI_KEY` environment variable
+            .baseUrl(System.getenv("AZURE_OPENAI_ENDPOINT"))
+            // Set the Azure Entra ID
+            .credential(BearerTokenCredential.create(AuthenticationUtil.getBearerTokenSupplier(
+                new DefaultAzureCredentialBuilder().build(), "https://cognitiveservices.azure.com/.default")))
+            .build();
+
+        String base64Image = null;
+        try {
+            String fileName = "logo.png";
+            byte[] imageBytes = Files.readAllBytes(Paths.get("src/samples/java/com/azure/ai/openai/stainless/resources/" + fileName));
+            base64Image = Base64.getEncoder().encodeToString(imageBytes);
+        } catch (IOException e) {
+            return;
```

Log and rethrow the exception instead of returning silently.

---

Done — the exception is now logged and rethrown. Appreciate the feedback!